src/plugins/intel_npu/src/plugin/npuw/partitioning/partitioning.cpp

```diff
@@ -1432,7 +1436,7 @@ void Partitioner::saveRepeatedConstants(const std::string& func_name) {
 }
 
 void Partitioner::saveTailDictConstants(const std::string& func_name) {
-    if (!cfg.get<::intel_npu::NPUW_HOST_GATHER_QUANT>()) {
+    if (!m_use_host_gather_quant) {
```

Comment:
let's have it controlled via `m_cfg` now - that looks more solid in this version

Reply:
As discussed, using context now